README.yaml

```
@@ -0,0 +1,111 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-rds-replica
+
+# Tags of this project
+tags:
+  - aws
+  - terraform
+  - terraform-modules
+  - databases
+  - rds
+  - aws-rds
+  - postgres
+  - mysql
+
+# Categories of this project
+categories:
+  - terraform-modules/databases
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-rds-replica
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-rds-replica.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-rds-replica"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-rds-replica.svg"
+    url: "https://github.com/cloudposse/terraform-aws-rds-replica/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+  - name: "terraform-aws-rds"
+    description: "Terraform module to provision AWS RDS instances for MySQL or Postgres"
+    url: "https://github.com/cloudposse/terraform-aws-rds"
+  - name: "terraform-aws-rds-cluster"
+    description: "Terraform module to provision an RDS Aurora cluster for MySQL or Postgres"
+    url: "https://github.com/cloudposse/terraform-aws-rds-cluster"
+  - name: "terraform-aws-rds-cloudwatch-sns-alarms"
+    description: "Terraform module that configures important RDS alerts using CloudWatch and sends them to an SNS topic"
+    url: "https://github.com/cloudposse/terraform-aws-rds-cloudwatch-sns-alarms"
+
+# Short description of this project
+description: |-
+  Terraform module to provision AWS [`RDS`](https://aws.amazon.com/rds/) replica instances. These are best suited for reporting purposes.
+
+  **IMPORTANT** It is not possible to create a read replica for a DB Instance that belongs to a DB Cluster.
```

```suggestion
  **IMPORTANT** It is not possible to create a read replica for a DB Instance that belongs to an Aurora Postgres Cluster.
```

Is this related to `Q: What kind of replicas does Aurora support?` in https://aws.amazon.com/rds/aurora/faqs/ ?

---

This is because I tried to use this to add a read replica to an instance created by the `terraform-aws-rds-cluster` module. It errored. I'm now implementing the same module for RDS cluster.